### DISCUSSION TITLE
Dual WAN UDM setup

### DISCUSSION THREAD

**Discussion by TuftyBruno** [unifi-utilities/unifi-common#487] [General]
I recently added a 2nd WAN to my UDM (in balanced mode) - NextDNS runs on UDM.  What is the best way to have the traffic routed through that WAN also filtered etc.?  I had previously done a ip4 assignment, but been reading that a DoH may be an option but kina lost there.  Also, noted that creating a 2nd profile and replicating settings (but that seem clunky since I have tons of rewrites.

Your advice is appreciated.

**Comment by boostchicken** [comment]
look in the 10-dns.sh file  It has iptables rules that will show you how you can force reqs to any place
https://github.com/unifi-utilities/unifios-utilities/blob/main/dns-common/on_boot.d/10-dns.sh

If you steal that script as it is and just plugin values for your setup (lots documentation in comments) you can have it set however you like, also this will  startup nextdns for you as well.
